examples/echo_bot.py

```diff
@@ -2,6 +2,7 @@
 from pyAitu import Bot, Dispatcher, executor
 from pyAitu.models import Message, QuickButtonSelected, InlineCommandSelected, ContentType,\
     QuickButtonCommand, InlineCommand, ReplyCommand, Media
+from pyAitu.models.media.contact import Contact
```

Comment:
please import only from models, it means define Contact in media.__init__ and models.__init__

Reply:
fixed